helm/hpcc/templates/dafilesrv.yaml

```diff
@@ -53,6 +53,7 @@ spec:
         helmVersion: 9.2.33-closedown0
       annotations:
         checksum/config: {{ $configSHA }}
+        {{- include "hpcc.generateAnnotations" $commonCtx | indent 8 }}
```

Comment:
It is a bit strange using indent (rather than nindent) in an indented block.  It works if the block leads with a newline, but would cause problems if not.

Reply:
using nindent would result in a newline here in cases where there was no content generated